### PR TITLE
[7.7.x] AF-1028 : [Project Oriented] Standalone perspectives are broken (#714)

### PR DESCRIPTION
**Pull request by Rikkola**
(cherry picked from commit 36dfae1801fa58ab4c137d753188c12c78786037)

https://github.com/kiegroup/kie-wb-common/pull/1854
https://github.com/kiegroup/kie-wb-distributions/pull/766

### CODE REVIEW

**Comment by Rikkola** [comment]
Could be that I have lost the formatting setup again. Need to check what is going on.

**Comment by barboras7** [comment]
jenkins please retest this

File: kie-drools-wb-parent/kie-drools-wb-webapp/src/main/java/org/kie/workbench/drools/client/perspectives/DroolsAuthoringPerspective.java [hunk 1]
```diff
@@ -1,152 +1,166 @@
...
+    private String projectPathString;
+    private boolean projectEditorDisableBuild;
+
+    private final List<PlaceRequest> placesToClose = new ArrayList<PlaceRequest>();
 
     @PostConstruct
-    public void setup() {
-        docks.setup( PerspectiveIds.AUTHORING, new DefaultPlaceRequest( "org.kie.guvnor.explorer" ) );
+    public void init() {
+        explorerMode = ( ( Window.Location.getParameterMap().containsKey( "explorer_mode" ) ) ? Window.Location.getParameterMap().get( "explorer_mode" ).get( 0 ) : "" ).trim();
```
Comment (jhrcek) [inline kie-drools-wb-parent/kie-drools-wb-webapp/src/main/java/org/kie/workbench/drools/client/perspectives/DroolsAuthoringPerspective.java]: @Rikkola would you mind reformatting this PR, please, so we have consistent code style?

**Comment by Rikkola** [comment]
@jhrcek Sure was just updating before I have to leave.

So the reason was for old formats was just that the class was renamed. Thought I had the wrong setup again.

**Comment by Rikkola** [comment]
"Perspective Projects should be loaded".
Not sure what here is blocking Projects.

jenkins please retest this

**Comment by jhrcek** [comment]
@Rikkola I'm no longer on VPN but I'll bet my trunks that you'll need to cherry-pick [this fix](https://github.com/kiegroup/kie-wb-distributions/commit/9b66a2dffda875df73ac9bc0f7ab56446f6ef900) from master to make that selenium test pass again.

**Comment by jhrcek** [comment]
@Rikkola now I am on VPN and I confirm my previous comment. Please cherry-pick that fix to your PR and the tests should pass fine.

**Comment by Rikkola** [comment]
@jhrcek Thank you. Cherry picked the commit in.

**Comment by Rikkola** [comment]
Ok all green and ready to go in.

**Pull request merged**